bee-api/bee-rest-api/src/types/dtos.rs

```diff
@@ -352,13 +365,29 @@ impl<'de> serde::Deserialize<'de> for OutputDto {
             {
                 SimpleOutput::KIND => OutputDto::Simple(
                     SimpleOutputDto::deserialize(value)
-                        .map_err(|e| serde::de::Error::custom(format!("can not deserialize output: {}", e)))?,
+                        .map_err(|e| serde::de::Error::custom(format!("can not deserialize simple output: {}", e)))?,
                 ),
                 TreasuryOutput::KIND => OutputDto::Treasury(
                     TreasuryOutputDto::deserialize(value)
-                        .map_err(|e| serde::de::Error::custom(format!("can not deserialize output: {}", e)))?,
+                        .map_err(|e| serde::de::Error::custom(format!("can not deserialize treasury output: {}", e)))?,
```

Comment:
apparently "cannot" is better suited for writing english than "can not".

Reply:
fixed the grammer!